python/tskit/trees.py

```diff
@@ -2793,6 +2793,25 @@ def kc_distance(self, other, lambda_=0.0):
         """
         return self._ll_tree.get_kc_distance(other._ll_tree, lambda_)
 
+    def path_length(self, *args):
+
+        """
+        Returns the path length between two nodes
+        (i.e., the number of edges between two nodes in this tree).
```

Comment:
```suggestion
        (i.e., the number of edges between two nodes in this tree).
        If the two nodes have a most recent common ancestor, then this is defined as 
        ``tree.depth(u) + tree.depth(v) - 2 * tree.depth(tree.mrca(u, v))``. If the nodes 
        do not have an MRCA (i.e., they are in disconnected subtrees) the path length 
        is infinity.
        
        .. seealso:: See also the :meth:`.depth` method
      
```

Reply:
Here's a bit more detail for the docstring. You'll need to compile the docs to see if the cross linking works, I'm never sure when I've got RST syntax right!